apps/web/src/app/test-kyc/page.tsx

```
@@ -0,0 +1,150 @@
+              <Button onClick={() => simulateKYCUpdate('pending')} variant="outline">Test Pending</Button>
+              <Button onClick={() => simulateKYCUpdate('approved')} variant="outline">Test Approved</Button>
+              <Button onClick={() => simulateKYCUpdate('rejected')} variant="outline">Test Rejected</Button>
+            </div>
+          </Card>
+        </div>
+
+        <div className="mt-6">
+          <h2 className="text-lg font-semibold mb-4">Test Results</h2>
+          <div className="space-y-2">
+            {testResults.map((result, index) => (
+              <div key={index} className="flex items-center gap-2 p-2 bg-gray-50 rounded">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)